t/hive-init-config.t

```diff
@@ -145,6 +145,11 @@ BEGIN {
         }});
         with 'OpusVL::FB11::Role::Brain';
     }
+
+    package TestInit::Brain::WithService::Hat::TEST::service {
+        use Moose;
+        with 'OpusVL::FB11::Role::Hat';
+    }
```

Comment:
Does this do anything in the tests at the moment?

Reply:
Added this because I implemented a missing exception, i.e. the one that told me when I asked for a hat class that didn't exist. So it stops the tests dying.